docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md

```diff
@@ -54,17 +54,37 @@ If you don't have any available hardware that match this requirement in the clus
 
 #### Upgrade Cluster Command for Scale Up/Down
 
-##### With Hardware CSV File
-
-```bash
-eksctl anywhere upgrade cluster -f cluster.yaml --hardware-csv <hardware.csv>
-```
-
-##### Without Hardware CSV File
-
-```bash
-eksctl anywhere upgrade cluster -f cluster.yaml
-```
+1. eksctl CLI: To create a workload cluster with eksctl, run:
+
+    ##### With Hardware CSV File
+    
+    ```bash
+    eksctl anywhere upgrade cluster -f cluster.yaml --hardware-csv <hardware.csv>
```

Comment:
```suggestion
    eksctl anywhere upgrade cluster \
         # --hardware-csv <hardware.csv>  \  # uncomment to add more hardware
         -f cluster.yaml 
```
We could do that same thing here. Drop "With/Without Hardware CSV..." and use one example.